third_party/OWNERS

```diff
@@ -1,14 +1,11 @@
 # The OWNERS file is used by prow to automatically merge approved PRs.
 
 approvers:
```

Comment:
Do we want to add serving-writers to the approver group here? I think these are mainly the nightly updates...